src/plots/plot.ts

```diff
@@ -20,6 +20,20 @@ export module Plots {
   }
 }
 
+/**
+ * Lightweight version of Plots.PlotEntity. The field selection is missing, but can be deduced
+ * using the drawer and the validDatumIndex.
```

Comment:
Possibly give a description on how this is Lightweight in comparison to the original Plots.PlotEntity?  Also, consider making it a `type`